sample/build.gradle

```diff
@@ -77,8 +77,8 @@ android {
 
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
-    implementation 'com.android.support:appcompat-v7:27.1.1'
-    implementation 'com.android.support.constraint:constraint-layout:1.1.1'
+    implementation "com.android.support:appcompat-v7:$supportLibraryVersion"
+    implementation "com.android.support.constraint:constraint-layout:$constraintLayoutVersion"
     testImplementation 'junit:junit:4.12'
```

Comment:
You have to use double quotes while using property value inside. Otherwise you don't need to do that. I don't see the reason for this change. 

Do you think we should use double quotes in all gradle file strings? They also differ from each other.